package.json

```diff
@@ -1,53 +1,54 @@
-  "homepage": "https://github.com/brigade/react-waypoint",
-  "bugs": "https://github.com/brigade/react-waypoint/issues",
-  "scripts": {
-    "build-npm": "rimraf build/npm && babel src/ -d build/npm/",
-    "test": "./node_modules/.bin/eslint . --ext .js,.jsx && ./node_modules/.bin/karma start",
-    "prepublish": "npm run build-npm"
-  },
-  "author": "Brigade Engineering",
-  "license": "MIT",
-  "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0"
-  },
-  "devDependencies": {
-    "babel": "^6.0.0",
-    "babel-cli": "^6.0.0",
-    "babel-core": "^6.0.0",
-    "babel-loader": "^6.0.0",
-    "babel-plugin-add-module-exports": "^0.1.2",
-    "babel-plugin-transform-react-jsx": "^6.0.0",
-    "babel-preset-es2015": "^6.0.0",
-    "eslint": "^1.3.1",
-    "eslint-config-brigade": "^1.6.0",
-    "eslint-plugin-react": "^3.3.1",
-    "jasmine-core": "^2.1.3",
-    "karma": "^0.13.9",
-    "karma-chrome-launcher": "^0.2.0",
-    "karma-cli": "^0.1.0",
-    "karma-firefox-launcher": "^0.1.6",
-    "karma-jasmine": "^0.3.6",
-    "karma-webpack": "^1.7.0",
-    "react": "^15.0.0",
-    "react-dom": "^15.0.0",
-    "webpack": "^1.5.3",
-    "rimraf": "^2.5.2"
-  },
-  "keywords": [
-    "react",
-    "component",
-    "react-component",
-    "scroll",
-    "onscroll"
-  ]
-}
+{
+  "name": "react-waypoint",
+  "version": "3.0.0",
+  "description": "A React component to execute a function whenever you scroll to an element.",
+  "main": "build/npm/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/brigade/react-waypoint.git"
+  },
+  "homepage": "https://github.com/brigade/react-waypoint",
+  "bugs": "https://github.com/brigade/react-waypoint/issues",
+  "scripts": {
+    "build-npm": "rimraf build/npm && babel src/ -d build/npm/",
+    "test": "./node_modules/.bin/eslint . --ext .js,.jsx && ./node_modules/.bin/karma start",
+    "prepublish": "npm run build-npm"
+  },
+  "author": "Brigade Engineering",
+  "license": "MIT",
+  "peerDependencies": {
+    "react": "^0.14.0 || ^15.0.0",
+    "react-dom": "^0.14.0 || ^15.0.0"
+  },
+  "devDependencies": {
+    "babel": "^6.0.0",
+    "babel-cli": "^6.0.0",
+    "babel-core": "^6.0.0",
+    "babel-loader": "^6.0.0",
+    "babel-plugin-add-module-exports": "^0.1.2",
+    "babel-plugin-transform-react-jsx": "^6.0.0",
+    "babel-preset-es2015": "^6.0.0",
+    "eslint": "^1.3.1",
+    "eslint-config-brigade": "^1.6.0",
+    "eslint-plugin-react": "^3.3.1",
+    "jasmine-core": "^2.1.3",
+    "karma": "^0.13.9",
+    "karma-chrome-launcher": "^0.2.0",
+    "karma-cli": "^0.1.0",
+    "karma-firefox-launcher": "^0.1.6",
+    "karma-jasmine": "^0.3.6",
+    "karma-webpack": "^1.7.0",
+    "lodash.throttle": "^4.0.1",
+    "react": "^15.0.0",
+    "react-dom": "^15.0.0",
+    "rimraf": "^2.5.2",
+    "webpack": "^1.5.3"
+  },
+  "keywords": [
+    "react",
+    "component",
+    "react-component",
+    "scroll",
+    "onscroll"
+  ]
+}
```

Comment:
Sorry for that!! :-( .. I don´t why.
Maybe cause I clonned on a windows machine.
I use spaces.